meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio_2.7.bbappend

```diff
@@ -17,3 +17,5 @@ SRC_URI:append:stm32mp1common = " \
 SRC_URI:append:sota:stm32mp1common = " \
      file://0003-FIO-interal-stm32mp1-support-for-boot-script-in-FIP.patch \
      "
+
+SRC_URI:append:stm32mp1common = " file://0001-Binutils-2.39-now-warns-when-a-segment-has-RXW-permi.patch"
```

Comment:
Why this is only for stm32mp1common?

Reply:
good question, maybe is not need anymore. I will check locally